File: BECore/app/model/MedicalRecord.js
/// <summary>
/// Model EXTJS MedicalRecord.
/// </summary>
Ext.define('QLDT.model.MedicalRecord', {
    extend: 'Ext.data.Model',
    idProperty: 'MedicalRecordID',
    idType: 'Guid',
    fields: [	
		{ name:"MedicalRecordID", type:"int" },
        { name: "CustomerID", type: "int" },
        { name: "CustomerName", type: "string" },
		{ name:"AppliedStandardID", type:"int" },
        { name:"MedicalRecordDescription", type:"string" },
		{ name:"MedicalRecordLocation", type:"string" },
        { name: "MedicalRecordDate", type:"date" },
		{ name:"FinalResult", type:"string" },
		{ name:"CreatedBy", type:"string" },
		{ name:"ModifiedBy", type:"string" },
		{ name:"IPAddress", type:"string" },
		{ name:"CreatedDate", type:"date" },
		{ name:"ModifiedDate", type:"date" },
     ]
});

File: MT.Model/HTTTYT/MedicalRecord.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region MedicalRecord:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a MedicalRecord:EntityBase.
    /// </summary>
    public class MedicalRecord : EntityBase
    {
        [Key]
        public int MedicalRecordID { get; set; }
        public int CustomerID { get; set; }
        public string CustomerName { get; set; }
        public int AppliedStandardID { get; set; }
        public string MedicalRecordDescription { get; set; }
        public string MedicalRecordLocation { get; set; }
        public DateTime MedicalRecordDate { get; set; }
        public string FinalResult { get; set; }


    }
    #endregion
}

